docs/process-definition/advisors.md

```diff
@@ -0,0 +1,15 @@
+[OWNERS]: ../../OWNERS
+[incubation]: incubation.md#main-stability
+
+# Dashboard Advisors
+
+Those considered "Advisors" of the Dashboard are also [OWNERS] file `approvers`.
+
+Two major responsibilities fall upon the Advisors.
+
+1. The first is they are the `approvers` of all PRs that go into the Dashboard - no code change can be done without an Advisor approval
+2. The second is less concrete; the Advisors help direct the path of the Dashboard code... they are intended to answer these questions:
+   - Want a community based feature?
```

Comment:
I'm not entirely sure what this question is asking?

Reply:
Improved wording and more clarity on the roles.